x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/predicate/operator/arithmetic/Add.java

```diff
@@ -30,4 +30,9 @@ protected Add replaceChildren(Expression left, Expression right) {
     public Add swapLeftAndRight() {
         return new Add(source(), right(), left());
     }
+
+    @Override
+    public ArithmeticOperation inverse(Source source, Expression left, Expression right) {
```

Comment:
return `Sub` instead of `ArithmeticOperation`

Reply:
Fixed.